CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ All notable changes to this project will be documented in this file based on the
 
 ### Breaking changes
 
+* Rename `event.raw` to `log.message`. #3
```

Comment:
This change should be removed now that `event.raw` stays as is.

Reply:
fixed, thanks for the review.